tutorials/classical_simulation/Simulating with effective noise channels.ipynb

```diff
@@ -15,25 +15,23 @@
     "\n",
```

Comment:
<p>Ah that's weird I thought I had modified this... In the end I don't think I will speak about probabilities - because I think the formula $`p(t) = \frac{1 - e^{-\gamma t}}{2}$ is true for dephasing noise, but is not for depolarizing noise - I don't think we have a formula for the depolarizing probability function of time @dehond ?</p>

Reply:
![image](https://github.com/pasqal-io/Pulser/assets/61453516/b8612c56-1a09-4ffa-88ba-6f09080c3cf8)

Reply:
![image](https://github.com/pasqal-io/Pulser/assets/61453516/1131aeb0-f74a-4326-a9e4-989fc492d8f7)

Reply:
> I don't think we have a formula for the depolarizing probability function of time @dehond ?

I suppose one could describe depolarizing noise as a decay envelope towards some stationary value, but this is only true in some limiting case (the same applies to dephasing noise). The expression given here is the decay of the contrast envelope for a single-atom Ramsey experiment. When interactions are introduced in a system this becomes more complicated, and in general it's not possible to describe it with such a clean analytical expression.